scripts/requirements.txt

```diff
@@ -0,0 +1,5 @@
+gym
+numpy
+torch == 0.4.1
```

Comment:
Is there a reason we are using 0.4.1 instead of 1.0.0?

Reply:
Thanks, applied your feedback.